configure.ac

```diff
@@ -197,13 +196,14 @@ AC_SUBST(MATE_ABOUT_CFLAGS)
 AC_SUBST(MATE_ABOUT_LIBS)
 
 dnl Language Support
+AM_GNU_GETTEXT([external])
+AM_GNU_GETTEXT_VERSION([0.19.8])
 
 GETTEXT_PACKAGE=mate-desktop
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE",
 		   [The gettext translation domain])
 AC_SUBST(GETTEXT_PACKAGE)
-
-AM_GLIB_GNU_GETTEXT
+AC_PATH_PROG([ITSTOOL], [itstool], [notfound])
```

Comment:
This line can be removed.
```shell
$ git grep ITSTOOL
configure.ac:AC_PATH_PROG([ITSTOOL], [itstool], [notfound])
```

Reply:
> > Where is this code called in user-admin?
> 
> ```
> [rave@mother user-admin]$ grep -nr mate_get_all_locales *
> src/user-language.c:440:    locale_ids = mate_get_all_locales ();
> ```
> 
> > I assume this is an autotools-built rpm?
> 
> Yes.
> Well, i don't get why port to getext have an influence here, but i biscet mate-desktop between 1.20.0 and latest master. and the result was this commit.
> I assume this is a hidden side effect of using gettext ?
> Maybe the ISO_CODES_PREFIX is ignored?
> [c84dd31#diff-0f600554fee93647481826d612ec562eR48](https://github.com/mate-desktop/mate-desktop/commit/c84dd31b4450f91bd8398353a66a873806ebb6ff#diff-0f600554fee93647481826d612ec562eR48)

Shouldn't, as far as I can tell its unconditionally applied in libmate-desktop/Makefile.am
and configure.ac. Do me a favor: `pkg-config  --variable prefix iso-codes`, if this is coming
up as anything other than `/usr` there may be a problem.

> In user-admin report we debugged the issue and the call from `mate_get_all_locales` to `usr/share/locale` did not work.
Can you strace the program and gist the log?

Reply:
I think you're right, i tested it and it seems it is ok to drop build requires itstool.
We don't ship a help pages here.
@hanetzer 
Or why did you add it?